halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/account/AbstractEditAccountCommand.java

```diff
@@ -58,6 +58,12 @@
   )
   List<String> requiredGroupMembership;
 
+  @Parameter(
+      names = "--bootstrap-only",
+      description = AccountCommandProperties.BOOTSTRAP_ONLY
+  )
+  Boolean bootstrapOnly = false;
```

Comment:
Don't set a default here, otherwise any edit that doesn't include this flag will turn off the --bootstrap-only mode.